app/assets/stylesheets/modules/_body.scss

```diff
@@ -2,3 +2,36 @@
   padding: 3em 0;
   margin-top: 2em;
 }
+
+
+#chart-container {
+  #benchmark-types-form-show {
+    padding-bottom: 8px;
+    padding-top: 8px;
+  }
+
+  .alert-warning {
+    padding: 8px;
+  }
+}
+
+@media (max-width: 547px) {
+  #chart-container {
+    #benchmark-types-form-show {
+      margin-bottom: 10px;
+      width: 100%;
+    }
+  }
+}
+
+@media (min-width: 548px) {
+  #chart-container {
+    #benchmark-types-form-show {
+      float: left;
+    }
+
+    .alert-warning-moved-right {
+      margin-left: 245px;
```

Comment:
hmm hacking in the margin here doesn't seem right to me. Could you see if there is another way to do it by setting the `columns` it takes up? Thanks!!